github/PullRequest.py

```diff
@@ -488,26 +496,34 @@ def is_merged(self):
         :calls: `GET /repos/:owner/:repo/pulls/:number/merge <http://developer.github.com/v3/pulls>`_
         :rtype: bool
         """
+        github._api_preview = True
         status, headers, data = self._requester.requestJson(
             "GET",
-            self.url + "/merge"
+            self.url + "/merge",
+            headers={'Accept': 'application/vnd.github.polaris-preview+json'}
         )
         return status == 204
 
-    def merge(self, commit_message=github.GithubObject.NotSet):
+    def merge(self, commit_message=github.GithubObject.NotSet, squash=github.GithubObject.NotSet):
         """
         :calls: `PUT /repos/:owner/:repo/pulls/:number/merge <http://developer.github.com/v3/pulls>`_
         :param commit_message: string
+        :param squash: boolean
         :rtype: :class:`github.PullRequestMergeStatus.PullRequestMergeStatus`
         """
+        squash = True
```

Comment:
Since you always set it to True right here, isn't it impossible for `squash` to be NotSet in the `if` statement below?